metrics/metrics.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	MOutgoingMsgs = stats.Int64("pubsub/outgoing_msg_size", "Outgoing message size", stats.UnitBytes)
```

Comment:
Since the measure is already package namespaced, it doesn't add much value to prefix with `M`.